### PR TITLE
feat: TE7-T1022: Implement theme change for home screen using mmkv

### PR DESCRIPTION
**Pull request by jigar-s-simform**
Implement theme change for home screen using mmkv for settings, videos and notifications screens

### CODE REVIEW

File: app/modules/notifications/NotificationsEmptyComponent.tsx [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
 import { Text, View } from 'react-native';
 import { Strings } from '../../constants';
-import { ThemeContext, ThemeType } from '../../context';
+import { ThemeType } from '../../context';
 import stylesheet from './NotificationStyles';
+import useNotification from './useNotification';
 
 const NotificationsEmptyComponent: FC = () => {
-  const {theme} = useContext(ThemeContext);
+  const { theme } = useNotification();
```
Comment (sagar-simformsolutions) [inline app/modules/notifications/NotificationsEmptyComponent.tsx]: add types

File: app/modules/notifications/useNotification.ts [hunk 2]
```diff
@@ -1,16 +1,24 @@
...
 export interface UseNotificationReturnType {
-    handleNotificationDelete: (notification: NotificationType) => void
+  handleNotificationDelete: (notification: NotificationType) => void;
+  theme: ThemeType;
 }
 
 const useNotification = (): UseNotificationReturnType => {
   const dispatch = useAppDispatch();
+  const appearance: ColorSchemeName = useColorScheme();
+  const [mmkvTheme] = useMMKVString(
```
Comment (sagar-simformsolutions) [inline app/modules/notifications/useNotification.ts]: add types

File: app/modules/settings/BottomSheetButton.tsx [hunk 3]
```diff
@@ -23,8 +24,10 @@ const BottomSheetButton = ({
   setSelected,
 }: BottomSheetButtonProps) => {
 
-  const { theme }: ThemeValueInterface =
-    useContext<ThemeValueInterface>(ThemeContext);
+  const appearance: ColorSchemeName = useColorScheme();
+  const [mmkvTheme] = useMMKVString(StorageConstants.themeStorageKey, mmkvStorage);
```
Comment (sagar-simformsolutions) [inline app/modules/settings/BottomSheetButton.tsx]: add types

File: app/modules/settings/WebViewComponent.tsx [hunk 4]
```diff
@@ -1,19 +1,26 @@
...
-import { ThemeContext } from '../../context';
+import { StorageConstants, Strings } from '../../constants';
+import { ThemeType, mmkvStorage } from '../../services';
 import stylesheet from './SettingsStyles';
 
 const WebViewComponent: FC = () => {
-    const { theme } = useContext(ThemeContext);
-    const styles = stylesheet(theme);
+  const appearance: ColorSchemeName = useColorScheme();
+  const [mmkvTheme] = useMMKVString(
```
Comment (sagar-simformsolutions) [inline app/modules/settings/WebViewComponent.tsx]: add types

File: app/modules/settings/useSettings.ts [hunk 5]
```diff
@@ -44,10 +46,9 @@ const useSettings = (
 ): UseSettingsReturnType => {
   const dispatch = useAppDispatch();
   const { userDetails } = useAppSelector(authSelector);
-  const sheetRef = useRef<BottomSheet>(null);
-  const appearance = useColorScheme();
-
-  const [_, setMmkvTheme] = useMMKVString(
+  const sheetRef: React.RefObject<BottomSheetMethods> = useRef<BottomSheet>(null);
+  const appearance: ColorSchemeName = useColorScheme();
+  const [mmkvTheme, setMmkvTheme] = useMMKVString(
```
Comment (sagar-simformsolutions) [inline app/modules/settings/useSettings.ts]: add types

**Review by sagar-simformsolutions — CHANGES_REQUESTED** [review]

File: app/modules/notifications/NotificationsEmptyComponent.tsx (hunk 1 above)
Comment (jigar-s-simform) [reply app/modules/notifications/NotificationsEmptyComponent.tsx]: resolved

File: app/modules/notifications/useNotification.ts (hunk 2 above)
Comment (jigar-s-simform) [reply app/modules/notifications/useNotification.ts]: resolved

File: app/modules/settings/BottomSheetButton.tsx (hunk 3 above)
Comment (jigar-s-simform) [reply app/modules/settings/BottomSheetButton.tsx]: resolved

File: app/modules/settings/WebViewComponent.tsx (hunk 4 above)
Comment (jigar-s-simform) [reply app/modules/settings/WebViewComponent.tsx]: resolved

File: app/modules/settings/useSettings.ts (hunk 5 above)
Comment (jigar-s-simform) [reply app/modules/settings/useSettings.ts]: resolved

**Review by sagar-simformsolutions — APPROVED** [review]

**Pull request merged**